city_metrix/layers/impervious_surface.py

```diff
@@ -0,0 +1,23 @@
+from dask.diagnostics import ProgressBar
+import xarray as xr
+import xee
+import ee
+
+from .layer import Layer, get_utm_zone_epsg, get_image_collection
+
+
+class ImperviousSurface(Layer):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    def get_data(self, bbox):
+        # load impervious_surface
+        dataset = ee.Image("Tsinghua/FROM-GLC/GAIA/v10").gt(0)  # change_year_index is zero if permeable as of 2018
+        imperv_surf = ee.ImageCollection(dataset
```

Comment:
`dataset` needs to be forced to ImageCollection to apply the `filterBounds()` and other functions.

`AttributeError: 'Image' object has no attribute 'filterBounds'`

Reply:
Forced dataset to be imagecollection, and added test for impervious surface. (Not 100% sure I did that right.)